disorder-core/disorder-core.cabal

```diff
@@ -17,6 +17,7 @@ library
                      , text                            >= 1.1        && < 1.3
                      , process                         == 1.2.*
                      , QuickCheck                      == 2.7.*
+                     , ieee754                         == 0.7.*
```

Comment:
> I'm unsure; there are no interface changes in 0.7.* to Data.AEq that I can see

Maybe I wasn't clear but that's what I was saying. This looks pretty stable. I just wanted to check since disorder is used everywhere.